packages/@aws-cdk/aws-rds/lib/props.ts

```diff
@@ -91,6 +91,13 @@ export interface InstanceProps {
    * @default - `true` if `vpcSubnets` is `subnetType: SubnetType.PUBLIC`, `false` otherwise
    */
   readonly publiclyAccessible?: boolean;
+
+  /**
+   * Indicates whether to copy tags from the DB instance to snapshots of the DB instance.
+   *
+   * @default: false
+   */
+  readonly copyTagsToSnapshot?: boolean;
```

Comment:
I see the following line the [documentation for this feature](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-rds-database-instance.html#cfn-rds-dbinstance-copytagstosnapshot):

> **Amazon Aurora**
>
> Not applicable. Copying tags to snapshots is managed by the DB cluster. Setting this value for an Aurora DB instance has no effect on the DB cluster setting.

So looks like this isn't applicable to Clusters, only to Instances? And Clusters have a separate setting?

Can you try deploying a Cluster with this property in `InstanceProps` set to `true`, and see whether the RDS service respects it?